e2e-network/expect-invoke-tls.sh

```diff
@@ -17,8 +17,8 @@ echo "[testing] $label"
 
 response="$(
   docker exec "$cli" peer chaincode invoke \
-    --peerAddresses "$peer:7051" \
-    --tlsRootCertFiles "/var/hyperledger/cli/crypto/peers/$peer/tls/ca.crt" \
+    --peerAddresses "$peer" \
+    --tlsRootCertFiles "/var/hyperledger/cli/crypto/peers/${peer%?????}/tls/ca.crt" \
```

Comment:
`?????`?

Reply:
takie coś `${peer%?????}` sprawi że zniknie 5 ostatnich znaków ze zmiennej peera. czyli `:port` który od teraz trzeba podawać w testach. zrobiłem to żeby trochę wygodniej i spójniej czytało się w testach e2e a nie port jako osobna zmienna.